draft-ietf-oauth-v2-1.md

```diff
@@ -1420,16 +1616,11 @@ sending the following HTTP response:
     HTTP/1.1 302 Found
     Location: https://client.example.com/cb?error=access_denied&state=xyz
 
+### Token Endpoint Extension {#code-token-extension}
 
-### Access Token Request {#access-token-request}
-
-The client makes a request to the token endpoint by sending the
-following parameters using the `application/x-www-form-urlencoded`
-format per Appendix B with a character encoding of UTF-8 in the HTTP
-request payload:
+The authorization grant type is identified at the token endpoint with the `grant_type` value of `authorization_code`.
 
-"grant_type":
-:    REQUIRED.  Value MUST be set to `authorization_code`.
+If this value is set, the following additional token request parameters beyond {{token-request}} are required:
```

Comment:
I'm not opposed to repeating all the parameters in this section, but currently it says only the additional parameters are defined but `client_id` is still in there.

Reply:
wasn't intentional - removed client_id